cli/lightning-cli.c

```diff
@@ -218,6 +218,7 @@ static void human_help(char *buffer, const jsmntok_t *result)
 	tal_free(help);
 
 	printf("---\nrun `lightning-cli help <command>` for more information on a specific command\n");
+        printf("\nExample: \n  lightning-cli invoice amount_msat=300 label=IV1 description=\"my first invoice\" expiry=300 \n---\n");
```

Comment:
This looks wrong!

```
➜  ~ clightning --testnet invoice amount_msat=300 label=IV1 description=\"my first invoice\" expiry=300
lightning-cli: Expected key=value in 'first': No such file or directory
```

```suggestion
        printf("\nExample: \n  lightning-cli -k invoice amount_msat=300 label=IV1 description=\"my first invoice\" expiry=300 \n---\n");
```

Reply:
What is printed by printf is without the \ in `description`:

    invoice amount_msat=300 label=IV1 description="my first invoice" expiry=300

that's why it's complaining about "first".

It could also be added to `lightning-cli help help` if it's to annoying.